tools/cipd/android_embedding_bundle/README.md

```diff
@@ -1,12 +1,34 @@
 # Updating the Embedding Dependencies
 
+The instructions in this README explain how to create a CIPD package that
+contains the build-time dependencies of the Android embedding of the Engine,
+and the dependencies of the in-tree testing framework. The Android embedder is
+shipped to Flutter end-users, but these build-time dependencies are not.
+Therefore, the license script can skip over the destination of the CIPD package
+in an Engine checkout at `src/third_party/android_embedding_dependencies`.
+Even so, the CIPD package should contain a LICENSE file, and the instructions
+below explain how to fetch the license information for the dependencies.
+
 ## Requirements
 
-1. Gradle. If you don't have Gradle installed, you can get it on [https://gradle.org/install/#manually](https://gradle.org/install/#manually).
-2. [Depot tools](http://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up).
+1. If you have a flutter/engine checkout, then you should already have
+[Depot tools](http://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up) on your path.
+1. You should have a copy of `gradle` in a flutter/engine checkout under
+   `src/third_party/gradle/bin/gradle`.
 
 ## Steps
 
-1. Update tools/androidx/files.json. (This file includes the Maven dependencies used to build Flutter apps).
-2. `cd` into this directory.
-3. Run `./generate.sh`.
+1. Update `tools/androidx/files.json`. (This file includes the Maven
+   dependencies used to build Flutter apps).
+1. `cd` into this directory.
```

Comment:
Can you spell out the directory? It is nice when following instructions that have not been run in a while to be super specific. 

```suggestion
1. `cd` into `tools/androidx` directory.
```

Reply:
Done.